public_html/wp-content/plugins/pattern-directory/includes/pattern-post-type.php

```diff
@@ -143,6 +143,25 @@ function register_post_type_data() {
 			),
 		)
 	);
+
+	register_post_meta(
```

Comment:
> I went with a meta field for now, since it can easily be set (in UI or WP-CLI) for the existing core patterns and doesn't need the full infrastructure of a taxonomy. 

🤔  At first glance, a taxonomy seems easier to me UI-wise, and has significant perf improvements for searching. I feel like I'm probably missing something, though?

Those benefits will probably be mitigated when we (presumably) switch to Elasticsearch, though. And you're right that we can always migrate if needed.

I'm open to either, just curious.

Reply:
We'll probably have to build our own UI for it regardless, or use components that will make no real difference (between sources). I also went with meta since we could use the registered block types as the list to choose from, while a taxonomy would have to be populated and kept up to date by a human.

Reply:
👍🏻